web/app/common/localStorage.js

```diff
@@ -0,0 +1,30 @@
+export const isAvailable = (() => {
```

Comment:
Why do you need check here, if you have it in constants.js?

Reply:
I thought of constant in "common/constants" as it defines global state of storage including cookies, localStorage, indexeddb and so on, it just happens that it the same as a constant in `common/localStorage`, but could be changed to more sophisticated version in future. I've changed for now, because maybe I'm over-complicating it.